### ISSUE TITLE
Use ImageMagick to auto-create GIFs of selected snapshots

### ISSUE THREAD

**Issue by jaypinho** [FrontPages/backend#14]
À la this: https://github.com/alykat/snowzilla-traffic

**Comment by jaypinho** [comment]
@jessepinho This could be a super-awesome tool for sharing on social media.

**Comment by jessepinho** [comment]
Sweet! This could cycle through all sites for a given moment or something?

**Comment by jaypinho** [comment]
Yup, exactly. (Actually, I called the model Collection, not Moment, but same thing.) And because there's a table called "collection_snapshots," each still image in the GIF could have its own caption, order, etc. too.
